lib/kbsecret/cli/kbsecret-login

```diff
@@ -37,6 +37,8 @@ selected_records = if cmd.opts.all?
                      end
                    end
 
+cmd.die "No such record." if selected_records.empty?
```

Comment:
This change should go in a separate PR (along with the `env` change I mentioned in https://github.com/kbsecret/kbsecret/issues/36#issuecomment-338429767).